public/embed-application.js

```diff
@@ -199,11 +223,22 @@ function attachSessionEventHandlers(embeddingApp) {
  */
 function attachErrorEventHandlers(embeddingApp) {
     embeddingApp.on("error", function(msg) {
+        console.error("ERROR: ", msg);
         switch (msg?.code?.toUpperCase()) {
             case "CONTAINER_MISSING":
                 // If the container to embed the content in cannot be found.
                 renderVisierErrorMessage("Invalid container", msg);
                 break;
+            case "FAILED_TO_EJECT_PREVIOUS_SESSION":
+                /**
+                 * When this is a second page that was opened that embeds content for a different user or tenant,
```

Comment:
If the user opens a second tab that embeds content for a different user or tenant, this error is emitted in the case of unexpected errors while performing cleanup on the first tab.